.generator/templates/api_doc.mustache

```diff
@@ -29,18 +29,18 @@ const configuration = {{{moduleName}}}.createConfiguration();
 const apiInstance = new {{{moduleName}}}.{{classname}}(configuration);
 
 {{#hasParams}}
-let body:{{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
+let params:{{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
 {{#allParams}}
   // {{{dataType}}}{{#description}} | {{{description}}}{{/description}}{{^required}} (optional){{/required}}
   {{paramName}}: {{{example}}},
 {{/allParams}}
 };
 {{/hasParams}}
 {{^hasParams}}
-let body:any = {};
+let params:any = {};
```

Comment:
Could we handle it to not defined any params when there is nothing to pass instead?

Reply:
We'll have to make the first argument (param: MonitorsApiGetMonitorRequest...) optional